BlueMap/Chat/src/main/resources/bluemap-chat.css

```diff
@@ -1,14 +1,22 @@
+/*TODO:
+   - Unable to select text while logged out.
+   */
+
 @font-face {
     font-family: "minecraft";
     src: url("./minecraft.otf") format("opentype");
 }
 
 #chat-root {
+    grid-auto-rows: 1fr auto;
+    display: grid;
+    grid-auto-columns: 1fr auto;
+    grid-gap: 10px;
     position: absolute;
     bottom: 0;
     left: 0;
     width: min(calc(100vw - 4em), 600px);
-    height: 300px;
+    max-height: min(calc(100vh - 4em), 800px);
```

Comment:
I'd say the max should be 50vh instead of 100vh - 4em, because on mobile devices the chat could fill up the whole screen

Reply:
Yes, the chat fills the whole screen. But that is why I had the toggle so that you chose when you can see the whole chat. When a message pops up for a short period of time, the map does not get covered. Maybe I can add a config option for this. Also, in testing, the chat never reaches the top bar.

Reply:
Ok fair, this can be left as-is